web/src/features/charts/bar-breakdown/elements/BySource.tsx

```diff
@@ -65,7 +66,14 @@ export default function BySource({
         <div className="flex gap-1">
           <h2>{text}</h2>
         </div>
-        {hasEstimationPill && <EstimationBadge text={pillText} />}
+        {hasEstimationPill && (
+          <EstimationBadge
+            text={pillText}
+            Icon={
+              estimationMethod === EstimationMethods.TSA ? CircleDashed : TrendingUpDown
+            }
+          />
+        )}
```

Comment:
Yes, since the top graph only shows one datetime we can't use the same logic in both. But I do think it's working as intended as all of the Turkey datetimes are estimated but only a few for Serbia.

It has the same behavior as now except that it changes the icon as well as the text instead of just the text.